backend/danswer/server/query_and_chat/query_backend.py

```diff
@@ -149,6 +150,7 @@ def get_answer_with_quote(
     query_request: DirectQARequest,
     user: User = Depends(current_user),
     db_session: Session = Depends(get_session),
+    _: bool = Depends(check_token_budget)
```

Comment:
I've just added it to this one route for now for testing. What other routes should this be applied to?

Reply:
Hmm, maybe also `send-message` (in `chat_backend.py`)

Reply:
Done!